model/execution/DeliveryExecutionManagerService.php

```diff
@@ -133,11 +156,13 @@ public function setExtraTime($deliveryExecutions, $extraTime = null)
 
             /** @var QtiTimer $timer */
             $timer = $this->getDeliveryTimer($deliveryExecution);
-            $timer->setExtraTime($extraTime)->save();
+            $timer->setExtraTime($extraTime);
+            $timer->setExtendedTime($extendedTime)->save();
```

Comment:
Why not to use `$timer->setExtraTime($extraTime)->setExtendedTime($extendedTime)->save();` ?